Orbital-Docs/_config.yml

```diff
@@ -18,30 +18,30 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
-title: Orbital
 email: opensource@focisolutions.com
 description: # this means to ignore newlines until "baseurl:"
-baseurl: '' # the subpath of your site, e.g. /blog
-url: '' # the base hostname & protocol for your site, e.g. http://example.com
-twitter_username: jekyllrb
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+title: ""
 github_username: FociSolutions
-minimal_mistakes_skin: 'contrast'
```

Comment:
I think your prettier is changing from double quotes to single. Can we change this to single?

Reply:
Changed!